packages/flutter/test/foundation/change_notifier_test.dart

```diff
@@ -282,7 +282,7 @@ void main() {
     final TestNotifier source3 = TestNotifier();
     final List<String> log = <String>[];
 
-    final Listenable merged = Listenable.merge(<Listenable>[source1, source2]);
+    final Listenable merged = Listenable.merge(<Listenable>{source1, source2});
```

Comment:
My preference would be to add a new test for this specific case so it is clear what we want to test. The test could be named "ensure that merging listenables works with non-lists" or something like that.